### PR TITLE
DDCE-6057 : Adding loggers

### PR DESCRIPTION
**Pull request by mithra-balakrishnan**
DDCE-6057 : Adding loggers

### CODE REVIEW

**Review by MRBeresford — COMMENTED** [review]
Just a few comments to make logs clearer.

File: app/services/ProcessCsvService.scala [hunk 1]
```diff
@@ -198,7 +141,7 @@ class ProcessCsvService @Inject()(auditEvents: AuditEvents,
       validator.validateRow(Row(0, getCells(parsedRow, 0)))
     } match {
       case Failure(exception) =>
-        logger.error(s"[ProcessCsvService][processRow] Exception returned when attempting to validate row: ${exception.getMessage}")
+        logger.error(s"[ProcessCsvService][processRow] Exception returned when attempting to validate row: ${exception.getMessage}", exception)
```
[the remark this answers is not in the document]
Comment (shridhar-chippa) [reply app/services/ProcessCsvService.scala:144]: Good to have

**Review by MRBeresford — APPROVED** [review]
LGTM

**Review by shridhar-chippa — APPROVED** [review]
LGTM

**Pull request merged**